meilisearch/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.18.0"
+__version__ = "0.18.1"
```

Comment:
Wouldn't this be 0.19.0 since a new feature was added?

Reply:
A new feature on `Meilisearch` and a new method on this SDK but if there are no breaking changes we just update as a minor.